### ISSUE TITLE
re-organize channels

### ISSUE THREAD

**Issue by jnsbck** [jaxleyverse/jaxley-mech#46]
add channels sorted by ions to readme.
make jx-mech.info() or similar that lists available channels and where they can be imported from + listed ions/properties/attrs.

**Comment by huangziwei** [comment]
yes, it'd be good to have a overview of all channels and synapses somewhere. 

I think we can implement different things for different purposes: 

1. just to list them somewhere: maybe add a `channels_and_synapses_info` notebook, and read all META data from them and list them in a data frame;
2. a function to find and return the specific channels in the code, e.g. `find_channel(name="Na")` and returns `[fm97.Na, benison01.Na, ...]` directly.

**Comment by huangziwei** [comment]
done #52
